ofproto/ofproto-dpif-xlate.c

```diff
@@ -3140,13 +3140,14 @@ get_fdb_data(struct xport *port, struct eth_addr mac_addr,
             if (!memcmp(smac.ea, mac_addr.ea, sizeof(smac))) {
                 VLOG_DBG("Ignore self MAC learn use case for port: %s",
                           port_name);
-                return -1;
+                //return -1;
```

Comment:
We should add a comment here to why we are changing the behavior

Reply:
Addressed